### ISSUE TITLE
rpmemd build failure on gcc 9

### ISSUE THREAD

**Issue by marcinslusarz** [pmem/issues#1002]
```
BUILDSTDERR: rpmemd_obc.c: In function 'rpmemd_obc_process':
BUILDSTDERR: rpmemd_obc.c:473:2: error: converting a packed 'struct rpmem_msg_hdr *' pointer (alignment 1) to 'struct rpmem_msg_hdr *' (alignment 8) may result in an unaligned pointer value [-Werror=address-of-packed-member]
BUILDSTDERR:   473 |  ret = rpmemd_obc_msg_recv(obc, &hdrp);
BUILDSTDERR:       |  ^~~
BUILDSTDERR: In file included from rpmemd_obc.c:49:
BUILDSTDERR: ../../../src/../src/rpmem_common/rpmem_proto.h:119:8: note: defined here
BUILDSTDERR:   119 
| struct rpmem_msg_hdr {
BUILDSTDERR:       |        ^~~~~~~~~~~~~
```

From https://apps.fedoraproject.org/koschei/build/5952412

**Comment by kilobyte** [comment]
Note that gcc-9 hasn't been released yet; rpmemd builds for me with gcc-9 20190102 (SVN revision 267505).

**Comment by marcinslusarz** [comment]
It was a false positive in prerelease version of gcc. Closing.
